inc/savedsearch.class.php

```diff
@@ -595,33 +584,19 @@ function getParameters($ID) {
       return false;
    }
 
-   /**
-    * Mark saved search as default view for the current user
-    *
-    * @param integer $ID ID of the saved search
-    *
-    * @deprecated since version 9.2; @see SavedSearch::markDefault()
-    *
-    * @return void
-   **/
-   function mark_default($ID) {
```

Comment:
This function only exist in bookmark.class in 9.1.5.
If you change class name, function can't be used even deprecated